rlberry/manager/evaluation.py

```diff
@@ -194,28 +196,30 @@ def read_writer_data(data_source, tag=None, preprocess_func=None, id_agent=None)
 
     if not isinstance(data_source, list):
         if isinstance(data_source, ExperimentManager):
-            data_source = [data_source]
+            experiment_manager_list = [data_source]
         else:
-            take_last_date = True
+            # find the path to the latest "manager_obj.pickle"
+            path_to_load = sorted(
+                pathlib.Path(data_source).glob("**/manager_obj.pickle"), reverse=True
+            )[0]
+            experiment_manager_list = [ExperimentManager.load(path_to_load)]
```

Comment:
Will there be a problem with this if you have several agents with different names saved into the same folder (i.e. same output_folder for two agents with different names) ?

Reply:
yes ... :(

Reply:
is that better if, instead using the timestep in the name, I use the "created date" of the file?
(if someone copy old (base on name) stuff, it may become the most recent stuff...)

Reply:
I think no because if there are several agents, you would still select only one while you wanted to have the last for each name. Maybe you can, before sorting, regexp to split into several lists, one for each name ?

Reply:
I don't know... another point of view could be:

If someone try different agent, and put the directory, maybe the best is to display only the last training (not the last training for all different agent).

If there are many agents to display, the user should use a list of directory (one path by agent).

No?